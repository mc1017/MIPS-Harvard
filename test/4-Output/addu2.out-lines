CPU : OUT :4294836247
