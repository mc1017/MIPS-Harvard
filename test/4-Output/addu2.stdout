CPU : OUT :4294836247
TB : Finished : active=0
test/1-ArithmeticandLS/addu2.v:50: $finish called at 76 (1s)
